website/templates/public/login.mako

```diff
@@ -42,34 +46,36 @@
             <div class="form-group" style="padding-top: 15px">
                 <div class="text-center m-t-lg">
                     %if redirect_url:
-                        <p>For non-institutional login, click <a href="/login/?redirect_url=${redirect_url}">here</a>.</p>
+                        <p>For non-institutional sign in, click <a href="/login/?redirect_url=${redirect_url}">here</a>.</p>
                     %else:
-                        <p>For non-institutional login, click <a href="/login/">here</a>.</p>
+                        <p>For non-institutional sign in, click <a href="/login/">here</a>.</p>
                     %endif
                 </div>
             </div>
         </div>
     </div>
+%endif
+%if campaign != "institution" or not enable_institutions:
+    %if existing_user:
+        <div class="col-sm-6 col-sm-offset-3 existing-user-signin p-b-md m-b-m">
```

Comment:
This is because the login page is centered in this view.

Before
---
![screen shot 2016-05-03 at 11 09 30 am]
(https://cloud.githubusercontent.com/assets/6232068/14997434/b33d6348-114c-11e6-96dc-f08e745d0cf8.png)

After
---
<img width="1608" alt="screen shot 2016-05-03 at 4 34 54 pm" src="https://cloud.githubusercontent.com/assets/6232068/14997514/07fbe3d2-114d-11e6-80d3-172d3563877f.png">